pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
   path: ^1.9.0
   riverpod: ^2.5.1
   riverpod_annotation: ^2.3.5
-  snapd: ^0.6.2
+  snapd: 0.6.4
```

Comment:
How come we need to pin this?

Reply:
We don't, I just forgot the caret when replacing my test branch with the new version number :)